web/web_test.go

```diff
@@ -19,3 +22,11 @@ func setupTestConfig() *Config {
 		Port: 80,
 	}
 }
+
+func newMockedSettingsService() services.SettingsService {
+	settigsService := new(services.MockSettingsService)
+
+	settigsService.On("InitializeIdentifier").Return(uuid.MustParse("59fd8017-b7fd-477b-9ebe-b658c558f3e9"), nil)
+
+	return settigsService
+}
```

Comment:
@dottorblaster take a look, we'll need the same approach for the premium/telemetry middleware, by just adding a `On` clause to the mock